chatops4s-discord/build.sbt

```diff
@@ -0,0 +1,23 @@
+ThisBuild / version := "0.1.0-SNAPSHOT"
+
+ThisBuild / scalaVersion := "3.3.6"
+
+lazy val root = (project in file("."))
+  .settings(
+    name := "chatops4s-discord",
+    libraryDependencies ++= Seq(
+      "com.softwaremill.sttp.tapir" %% "tapir-core"             % "1.11.34",
+      "com.softwaremill.sttp.tapir" %% "tapir-cats"             % "1.11.34",
+      "com.softwaremill.sttp.tapir" %% "tapir-http4s-server"    % "1.11.34",
```

Comment:
The end goal is for the client to select the backend. Let me elaborate on this during the sync call